appcues/src/main/java/com/appcues/trait/appcues/BackdropKeyholeTrait.kt

```diff
@@ -63,8 +63,10 @@ internal class BackdropKeyholeTrait(
         val density = LocalDensity.current
         val metadata = LocalAppcuesStepMetadata.current
 
+        // trait does not apply if no target rectangle in metadata
+        val targetRectInfo = rememberTargetRectangleInfo(metadata) ?: return
```

Comment:
can you go over in more depth what was the issue here in this trait?

Reply:
updated to guard `encompassesRadiusPx.value` must be > 0 instead on the `drawRoundRect` block, which I think is the appropriate check.